src/rss.cpp

```diff
@@ -302,17 +302,21 @@ void rss_item::set_flags(const std::string& ff) {
 void rss_item::sort_flags() {
 	std::sort(flags_.begin(), flags_.end());
 
-	for (auto it=flags_.begin(); flags_.size() > 0 && it!=flags_.end(); ++it) {
+	auto it=flags_.begin(); 
+	while (flags_.size() > 0 && it!=flags_.end()) {
 		if (!isalpha(*it)) {
 			flags_.erase(it);
 			it = flags_.begin();
```

Comment:
Why rewind to the beginning when we can simply carry on as-is? (After `erase` `it` will point to the next character.)